test/fops/file.c

```diff
@@ -51,6 +53,22 @@ obj_bulk(void)
 		} else if (ret != ENOENT && ret != EBUSY && ret != EINVAL)
 			testutil_die(ret, "session.open_cursor bulk");
 	}
+
+	if (use_txn) {
+		/*
+		 * As the operations are being performed concurrently,
+		 * return value can be ENOENT, EBUSY or EINVAL.
+		 * __wt_curfile_insert_check when returns error, will set
```

Comment:
Don't refer to internal library function names in application programs.  I would just remove that sentence.